internal/pkg/manifest/workload_ecs.go

```diff
@@ -183,7 +183,7 @@ func (t TaskConfig) IsARM() bool {
 
 // Secret represents an identifier for sensitive data stored in either SSM or SecretsManager.
 type Secret struct {
-	from               *string              // SSM Parameter name or ARN to a secret.
+	from               stringOrFromCFN      // SSM Parameter name or ARN to a secret.
```

Comment:
```suggestion
	from               stringOrFromCFN      // SSM Parameter name or ARN to a secret or secret name imported from another stack.
```

Reply:
+1 to Parag! A minor suggestion that I have is 
```
...// SSM Parameter name, plain secret ARN, or secret ARN imported from another stack.
```